CONTRIBUTING.md

```diff
@@ -136,6 +137,47 @@ cases to our [automated test suite](ci/README.md) to guarantee enough
 coverage. A PR that makes significant code changes without contributing new test
 cases will be flagged by reviewers and will not be accepted.
 
+### Getting your PR verified by CI
+
+It is a requirement to get your PR verified with CI checks before it gets merged. Also, it helps to find possible bugs
```

Comment:
could you wrap lines at ~80 chars?

Reply:
Updated.

Sorry, the vertical line in my IDE is misleading...